### ISSUE TITLE
Update many users Zendesk REST API option. Coming?

### ISSUE THREAD

**Issue by larssb** [blakmatrix/node-zendesk#103]
Hi creators of node-zendesk.....

I was wondering if there are any plans for implementing the Zendesk Update-many-users endpoint. It can be found here: https://developer.zendesk.com/rest_api/docs/core/users#update-many-users

It would be far easier than having to iterate over users and create them 1by1. Before then requesting the Zendesk REST API to create each user by individual requests. I hope you follow me, I will gladly elaborate further.

I see that you have it listed as not implemented yet: https://github.com/blakmatrix/node-zendesk/blob/master/doc/api-coverage.md - clearly telling me that you are aware. 

So basically a kind request for a status on the update endpoint :-)

Thank you sincerely and I'm looking forward to hear from you.
Kind regards
/Lars

**Comment by nnabuuu** [comment]
Yes we have that plan.. Well let's check (In the weekend maybe ( •̀ᄇ• ́)ﻭ✧) if we can have a quick implement for it and whether there is any blocking issue.

**Comment by larssb** [comment]
@nnabuuu fantastic. Looking forward to hear more from you guys. Yeaaah! :-D

**Comment by larssb** [comment]
@nnabuuu thank you....just downloaded the new files today. Looking forward to try it out. Do you know when a new NPM will be released or is this not considered for a incremental jump in version? 

Again thank you.

**Comment by nnabuuu** [comment]
Oh.. That thing.. I forgot as I never did any release it in NPM... Maybe @erikzrekz  and @blakmatrix will do it..

**Comment by erikzrekz** [comment]
@nnabuuu @larssb just bumped to v1.1.5 :+1:

**Comment by larssb** [comment]
Thank u...I've also tested/implemented the new way of updating many users. It works. So thnx! I think you can close this one.
